tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/CopyPortalSettingsHandler.java

```diff
@@ -38,11 +42,16 @@ public static final Status execute( final CopyPortalSettingsOp op, final Progres
 
         final Map<String, File> parameters = new HashMap<>();
 
-        parameters.put( "source", op.getSourceLiferayLocation().content().toFile() );
-        parameters.put( "dest", op.getDestinationLiferayLocation().content().toFile() );
-
         try
         {
+            UpgradeAssistantSettingsUtil.JavaObjectToJSONFile(
```

Comment:
This doesn't really work.

1. it doesn't check for current object that may already be stored.
2. it doesn't use the top level file Liferay7UpgradeAssistantSettings, this code is directly using PortalSettings